docs/community/exhibition-checklist.md

```diff
@@ -0,0 +1,116 @@
+Exhibition checklist
+====================
+
+This page is a checklist for important tasks for FOSS@MAGIC community members to follow before a public exhibition or show.
+On a semester-to-semester basis, this usually means the [Maker Faire Rochester](https://rochester.makerfaire.com/) and [Imagine RIT](https://www.rit.edu/imagine/).
+Occasionally, it may include a booth at a major computing conference, like [OSCON](https://fossrit.github.io/events/2017/05/10/oscon/).
+
+
+## Purpose
+
+Exhibitions, trade shows, and conferences are important opportunities to showcase our community to people outside of RIT.
+Around the year, students, faculty, and staff are doing cool things with open source.
+We might be familiar with it being on campus, but sharing our work with others is an important part of getting our work out there to the world.
+This also provides students a chance to show off some of their work to prospective employers or community leaders.
+
+Therefore, it is important to maintain a respectful and professional attitude for our exhibitions at different events.
+They are as much a chance to promote FOSS@MAGIC as they are to promote work and efforts by individual members of our community.
+
+
+## Staffing requirements
+
+**Minimum two people _must_ staff a booth/table during an exhibition.**
+Three people is recommended and preferred.
+
+Everyone who volunteers to staff a FOSS@MAGIC booth should also have time to go around the conference, go to talks or workshops, and of course use the bathroom.
+Keeping two to three people at the booth avoids one person taking everyone's questions and getting exhausted.
+Be kind to your fellow volunteers and try to be flexible so everyone has a chance to volunteer at the booth, but also experience the event as an attendee too.
+
+
+## Preparation checklist
+
+Sections below are in sequential order.
+
+### Publish call for projects on FOSS@MAGIC website
```

Comment:
Is there a location for where we store active FOSS@MAGIC projects? We often show off some usual long-running projects and it'd be good to not have to always re-submit them for each event in this sort of call thing.

Reply:
I assume this would be the website that we would store them but just being clear :)

Reply:
We do, but it needs a whole lot of love.

https://fossrit.github.io/projects/